ta/pkcs11/src/processing_ec.c

```diff
@@ -847,16 +854,20 @@ size_t ecdsa_get_input_max_byte_size(TEE_OperationHandle op)
 
 	TEE_GetOperationInfo(op, &info);
 
-	switch (info.algorithm) {
-	case TEE_ALG_ECDSA_P192:
+	/*
+	 *  Determining curve size in bytes with the help of
+	 *  maxkeysize attribute instead of algorithm
```

Comment:
ìnstead of algorithm`: this comment should rather be in the commit message, not in the source file.